packages/client/modules/demo/handleCompletedDemoStage.ts

```diff
@@ -61,9 +61,10 @@ const addStageToBotScript = (stageId: string, db: RetroDemoDB, reflectionGroupId
         }))
       )
     }
-    const comment = commentLookup[reflection.id]
-    if (comment) {
-      comments.push(comment)
+    const comment = commentLookup[reflection.id as keyof typeof commentLookup]
+    if (Array.isArray(comment)) {
+      // I think this was a bug before...
```

Comment:
noImplicitAny caught a bug here

Reply:
all the values in `commentLookup` are arrays. perhaps that was an error? i didn't want to change that though